Dataset Scheming Form Pages [OPEN-4034]

wardi opened the pull request:
split dataset type into separate pages using scheming form pages and dataset completeness feature

wardi committed:
OPEN-4034: form pages for dataset type

wardi committed:
changes

wardi committed:
allow scheming pages functionality to pass through

wardi committed:
one more is_resource_supported_by_xloader check

wardi committed:
allow scheming create redirect to page 2

RabiaSajjad committed:
improve display for the navigation for form stages

RabiaSajjad committed:
move resources snippet out of secondary block

JVickery-TBS committed:
Merge branch 'OPEN-4034-form-pages' into feature/temp/form-pages

# Conflicts:
#	ckanext/canada/templates/package/snippets/resources.html
#	ckanext/canada/templates/scheming/package/snippets/package_form.html
### RESOLVED.

JVickery-TBS committed:
Merge branch 'form-pages-ui' into feature/temp/form-pages

JVickery-TBS committed:
feat(dev): form pages;

- Cont form pages.

JVickery-TBS committed:
Merge branch 'form-pages-ui' into OPEN-4034-form-pages

JVickery-TBS committed:
Merge branch 'feature/temp/form-pages' into OPEN-4034-form-pages

JVickery-TBS commented:
Had to do some more css and template stuff, mainly for the Add Resource in Draft part. and then some small screen stuff.

Have to do a couple more to polish this up. But this is in staging and working fairly well now.

JVickery-TBS committed:
Merge branch 'master' into OPEN-4034-form-pages

JVickery-TBS committed:
Merge branch 'master' into OPEN-4034-form-pages

JVickery-TBS committed:
feat(dev): scheming pages;

- Cont scheming pages.

JVickery-TBS committed:
Merge branch 'master' into OPEN-4034-form-pages

JVickery-TBS committed:
feat(dev): scheming pages;

- Cont scheming pages.

JVickery-TBS committed:
Merge branch 'master' into OPEN-4034-form-pages

JVickery-TBS committed:
Merge branch 'master' into OPEN-4034-form-pages

JVickery-TBS committed:
Merge branch 'master' into OPEN-4034-form-pages

JVickery-TBS committed:
feat(templates) manage dropdown;

- Make the manage button for datasets dropdown with page links for scheming pages.

JVickery-TBS committed:
fix(syntax): typing;

- Pyright fixes.

wardi committed:
rough sidebar stages

JVickery-TBS committed:
feat(templates): sidebar stages cont;

- Additional layout changes for the sidebar stages/pages.

JVickery-TBS committed:
fix(templates): pd forms;

- Fix stages sidebar template call for pd types.
- Fix a11y issues in the webform pages.

wardi committed:
info schema with pages

JVickery-TBS committed:
Merge branch 'OPEN-4035-info-pages' into OPEN-4034-form-pages

wardi committed:
remove dataset stages side bar from resource pages

JVickery-TBS committed:
Merge branch 'master' into OPEN-4034-form-pages

# Conflicts:
#	ckanext/canada/assets/public/canada_public.css
### RESOLVED.

JVickery-TBS committed:
idk

JVickery-TBS committed:
tests ig

JVickery-TBS committed:
tests ig

JVickery-TBS committed:
Merge branch 'master' into OPEN-4034-form-pages

# Conflicts:
#	.github/workflows/pytest.yml
### RESOLVED.

JVickery-TBS committed:
fix(tests,schema): skip tests, validator orders;

- Put `scheming_required` after `fluent_text`
- Skipped some tests for now.

JVickery-TBS committed:
feat(template): resource stages sidebar;

- Show stages sidebar for Resource part when adding first resource.

JVickery-TBS committed:
Merge branch 'master' into OPEN-4034-form-pages

JVickery-TBS committed:
Merge branch 'master' into OPEN-4034-form-pages

JVickery-TBS committed:
feat(dev): configurable;

- Make the scheming pages configurable for use.

JVickery-TBS committed:
feat(dev): configurable;

- Make the scheming pages configurable for use.

JVickery-TBS committed:
Merge branch 'master' into OPEN-4034-form-pages

# Conflicts:
#	ckanext/canada/schemas/dataset.pages.yaml
#	ckanext/canada/schemas/info.pages.yaml
#	ckanext/canada/schemas/presets.pages.yaml
### RESOLVED

JVickery-TBS committed:
feat(dev): recreate;

- Recreate git changes.

JVickery-TBS committed:
feat(dev): recreate;

- Recreate git changes.

JVickery-TBS committed:
fix(lint): syntax;

- Pyright fixes.
- Flake8 fixes.

JVickery-TBS committed:
Merge branch 'master' into OPEN-4034-form-pages

# Conflicts:
#	ckanext/canada/schemas/presets.pages.yaml
### RESOLVED.
